src/components/content/Form.js

```diff
@@ -1,26 +1,65 @@
 import React, {Component} from 'react';
 import "bootstrap/dist/css/bootstrap.css";
+
 class Form extends Component {
+    handleSubmit = (event) => {
+        event.preventDefault();
+        const email = document.getElementById('inputEmail');
+        const pass = document.getElementById('inputPassword');
+        const rememberCheckBox = document.getElementById('check').getAttribute('checked');
+        const url = 'https://5b3643ec6005b00014c5dcf2.mockapi.io/users';
+        const status = (response) => {
+            if (response.status >= 200 && response.status < 300) {
+                return Promise.resolve(response)
+            } else {
+                console.log(response);
+                return Promise.reject(new Error(response.statusText))
+            }
+        };
+        const json = (response) => {
+            return response.json()
+        };
+
+        fetch(url, {
+            method: 'post',
+            headers: {
+                "Content-type": 'application/x-www-form-urlencoded'
+            },
+            body: `email=${email}&password=${pass}&remember-me=${rememberCheckBox}`
+        })
+            .then(status)
+            .then(json)
+            .then((data) => {
+                console.log('Request succeeded with JSON response', data);
```

Comment:
I think it is better to handle successful login in another way. Think about it. Maybe save login flag  in storage or smth like that. How will your app work if you reload page or browser?
So, you should thinnk about component composing or another way like through storage.
Although, these problems with  authorizing (sign in) are resolved on server side, but, plz, read how can we resolve this issue.